pkg/testutil/testutil.go

```diff
@@ -31,3 +37,21 @@ func MakeTestDir(t *testing.T, sourceDir string) string {
 
 	return tmpDir
 }
+
+// Assert that the contents of two files are equal.
+//
+// To update the expected file, run the test with PULUMI_ACCEPT=true.
+func AssertFilesEqual(t *testing.T, expectedPath, generatedPath string) {
```

Comment:
I'm not sure how test files were updated previously, but this is the standard way we do this kind of update.